### ISSUE TITLE
Generating tags from user input after initial object creation?

### ISSUE THREAD

**Issue by SilverStrings024** [knyghty/django-snakeoil#6]
I'm trying to allow my staff to add some SEO keywords by giving them a CharField then parsing that field in the snakeoil_metadata method. The object is originally created by visitors to the website and then the tags are added before approval by staff. There doesn't seem to be a way to do this, is there something that could allow me to do this that's already in the package or would I need to modify the package to allow this (which is fine)? If I do need to add something to add to this package, should I submit a PR to allow for this feature?

**Comment by knyghty** [comment]
@SilverStrings024 hi, I'm sorry I didn't reply to this at the time, I probably didn't notice because you closed the issue so quickly.

I just wanted to reply because I don't quite understand what you're asking and I just wanted to clarify if you found a way to do it?

**Comment by SilverStrings024** [comment]
@knyghty Hello, sorry for closing this so quickly. I was still fairly new to coding and didn't know how to really deal with github issues so, I apologize.
I don't remember what I did to fix this but I know for a fact I did; let me take a look through my old code base and see what I did and then I'll let you know. It's also possible that I switched packages because I remember using matomo (utilizing `django-analytical`) so it's possible but I'm sure it wouldn't be hard to allow this if you'd like a bit of help making this a thing if it isn't already!
